src/Microsoft.ML.Transforms/MissingValueDroppingTransformer.cs

```diff
@@ -13,16 +13,16 @@
 using Microsoft.ML.Runtime.Internal.Utilities;
 using Microsoft.ML.Runtime.Model;
 
-[assembly: LoadableClass(NADropTransform.Summary, typeof(NADropTransform), typeof(NADropTransform.Arguments), typeof(SignatureDataTransform),
-    NADropTransform.FriendlyName, NADropTransform.ShortName, "NADropTransform")]
+[assembly: LoadableClass(MissingValueDroppingTransformer.Summary, typeof(MissingValueDroppingTransformer), typeof(MissingValueDroppingTransformer.Arguments), typeof(SignatureDataTransform),
+    MissingValueDroppingTransformer.FriendlyName, MissingValueDroppingTransformer.ShortName, "NADropTransform")]
 
-[assembly: LoadableClass(NADropTransform.Summary, typeof(NADropTransform), null, typeof(SignatureLoadDataTransform),
-    NADropTransform.FriendlyName, NADropTransform.LoaderSignature)]
+[assembly: LoadableClass(MissingValueDroppingTransformer.Summary, typeof(MissingValueDroppingTransformer), null, typeof(SignatureLoadDataTransform),
+    MissingValueDroppingTransformer.FriendlyName, MissingValueDroppingTransformer.LoaderSignature)]
 
 namespace Microsoft.ML.Runtime.Data
 {
     /// <include file='doc.xml' path='doc/members/member[@name="NADrop"]'/>
```

Comment:
>NADrop [](start = 64, length = 6)

nit: these names can potentially backfire, but I'm don't know what our plans for them, and documentation in general.

Reply:
documentation needs to be scrubbed from the term transform, and adapted to transformer/estimator. Didn't want to bundle those changes together with the renames, because the renames have to finish on this release. 

---
In reply to: [232742005](https://github.com/dotnet/machinelearning/pull/1590#discussion_r232742005) [](ancestors = 232742005)